### PR TITLE
Add reload buttons to store and snapshot

### LINKED ISSUE #5698: Missing Refesh or reload snapshot button in 0.109.2
## Checklist

- [x ] I have updated to the latest available Home Assistant version.
- [x ] I have cleared the cache of my browser.
- [x ] I have tried a different browser to see if it is related to my browser.

## The problem

Missing Refesh or reload snapshot button in 0.109.2

## Expected behavior

the button was there before.

## Steps to reproduce

Go to snapshots tab

## Environment

- Home Assistant release with the issue: 0.109.2
- Last working Home Assistant release (if known): 0.108.8
- Browser and browser version: Firefox and Google Chrome
- Operating system: Ubuntu Desktop 18.04lts

## State of relevant entities
none
## Problem-relevant configuration
none
## Javascript errors shown in your browser console/inspector
none

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

This adds back the reload button to the store and snapshot pags.
image

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #5698
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/addon-store/hassio-addon-store.ts [hunk 1]
```diff
@@ -52,42 +59,61 @@ class HassioAddonStore extends LitElement {
...
     const repos: TemplateResult[] = [];
 
-    for (const repo of this._repos) {
-      const addons = this._addons!.filter(
-        (addon) => addon.repository === repo.slug
-      );
-
-      if (addons.length === 0) {
-        continue;
+    if (this._repos) {
```
Comment (bramkragten) [inline hassio/src/addon-store/hassio-addon-store.ts:64]: We should do this in a `memoizeOne`

File: hassio/src/addon-store/hassio-addon-store.ts [hunk 2]
```diff
@@ -52,42 +59,61 @@ class HassioAddonStore extends LitElement {
...
-      ></hassio-repositories-editor>
-
-      <hassio-search-input
-        .filter=${this._filter}
-        @value-changed=${this._filterChanged}
-      ></hassio-search-input>
-
-      ${repos}
+        .narrow=${this.narrow}
+        !hassio
```
Comment (bramkragten) [inline hassio/src/addon-store/hassio-addon-store.ts:89]: What's this?

Comment (ludeeus) [reply hassio/src/addon-store/hassio-addon-store.ts:89]: How the h... did that happen....

Comment (bramkragten) [reply hassio/src/addon-store/hassio-addon-store.ts:89]: Is the `!` a typo?

Comment (ludeeus) [reply hassio/src/addon-store/hassio-addon-store.ts:89]: The entire line should not be there, so I don't think so

File: hassio/src/dashboard/hassio-dashboard.ts [hunk 3]
```diff
@@ -13,34 +13,49 @@ import {
...
-          .supervisorInfo=${this.supervisorInfo}
-          .hassOsInfo=${this.hassOsInfo}
-        ></hassio-update>
-        <hassio-addons
-          .hass=${this.hass}
-          .addons=${this.supervisorInfo.addons}
-        ></hassio-addons>
-      </div>
+      <hass-tabs-subpage
+        .hass=${this.hass}
```
Comment (bramkragten) [inline hassio/src/dashboard/hassio-dashboard.ts:40]: `hassio`

File: hassio/src/hassio-panel.ts [hunk 4]
```diff
@@ -34,46 +56,16 @@ class HassioPanel extends LitElement {
...
-        path: `/hassio/system`,
-        icon: "mdi:cogs",
-      },
-    ];
-
     return html`
-      <hass-tabs-subpage
+      <hassio-panel-router
+        .route=${this.route}
         .hass=${this.hass}
```
Comment (bramkragten) [inline hassio/src/hassio-panel.ts:62]: `hassio`

File: hassio/src/snapshots/hassio-snapshots.ts [hunk 5]
```diff
@@ -81,135 +88,151 @@ class HassioSnapshots extends LitElement {
...
-                @click=${this._createSnapshot}
-              >
-                Create
-              </mwc-button>
-            </div>
-          </paper-card>
-        </div>
+      <hass-tabs-subpage
+        .hass=${this.hass}
+        .narrow=${this.narrow}
```
Comment (bramkragten) [inline hassio/src/snapshots/hassio-snapshots.ts:93]: `hassio`

File: hassio/src/system/hassio-system.ts [hunk 6]
```diff
@@ -32,22 +39,30 @@ class HassioSystem extends LitElement {
...
-            .supervisorInfo=${this.supervisorInfo}
-          ></hassio-supervisor-info>
-          <hassio-host-info
-            .hass=${this.hass}
-            .hostInfo=${this.hostInfo}
-            .hassOsInfo=${this.hassOsInfo}
-          ></hassio-host-info>
+      <hass-tabs-subpage
+        .hass=${this.hass}
+        .narrow=${this.narrow}
```
Comment (bramkragten) [inline hassio/src/system/hassio-system.ts:44]: `hassio`

**Pull request merged**